tests/baselines/reference/docker/azure-sdk.log

```diff
@@ -2,139 +2,141 @@ Exit Code: 1
-@azure/event-hubs (? seconds)
-@azure/event-processor-host (? seconds)
-@azure/storage-blob (? seconds)
-@azure/ai-text-analytics (? seconds)
-@azure/app-configuration (? seconds)
-@azure/core-arm (? seconds)
-@azure/core-tracing (? seconds)
-@azure/eventhubs-checkpointstore-blob (? seconds)
-@azure/identity (? seconds)
-@azure/search (? seconds)
-@azure/service-bus (? seconds)
-@azure/storage-file-datalake (? seconds)
-@azure/storage-file-share (? seconds)
-@azure/storage-queue (? seconds)
-@azure/template (? seconds)
-testhub (? seconds)
-================================
-SUCCESS WITH WARNINGS (1)
 ================================
-@azure/cosmos (? seconds)
-dist-esm/index.js → dist/index.js...
-(!) Unresolved dependencies
-https://rollupjs.org/guide/en/#warning-treating-module-as-external-dependency
-os-name (imported by dist-esm/utils/user-agent.js)
-(!) Missing global variable name
-Use output.globals to specify browser global variable names corresponding to external modules
-os-name (guessing 'osName')
-created dist/index.js in ?s
-================================
-BLOCKED (1)
+BLOCKED (36)
 ================================
+@azure/core-auth
+@azure/core-http
+@azure/core-lro
+@azure/dev-tool
+@azure/identity
+@azure/test-utils-recorder
+@azure/core-amqp
+@azure/core-https
+@azure/core-xml
+@azure/event-processor-host
+@azure/keyvault-secrets
+@azure/storage-blob
+@azure/ai-form-recognizer
+@azure/ai-text-analytics
+@azure/app-configuration
+@azure/core-arm
+@azure/core-client
+@azure/core-tracing
+@azure/cosmos
+@azure/event-hubs
+@azure/eventhubs-checkpointstore-blob
+@azure/keyvault-admin
 @azure/keyvault-certificates
+@azure/keyvault-common
+@azure/keyvault-keys
+@azure/search-documents
+@azure/service-bus
+@azure/storage-blob-changefeed
+@azure/storage-file-datalake
+@azure/storage-file-share
+@azure/storage-internal-avro
+@azure/storage-queue
+@azure/tables
+@azure/template
+@azure/test-utils-perfstress
+testhub
 ================================
 FAILURE (2)
 ================================
-@azure/keyvault-keys ( ? seconds)
-npm ERR! code ELIFECYCLE
-npm ERR! errno 2
-npm ERR! @azure/keyvault-keys@X.X.X-preview.2 extract-api: `tsc -p . && api-extractor run --local`
-npm ERR! Exit status 2
-npm ERR!
-npm ERR! Failed at the @azure/keyvault-keys@X.X.X-preview.2 extract-api script.
-npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
-npm ERR! A complete log of this run can be found in:
-npm ERR!     /root/.npm/_logs/XXXX-XX-XXXXXXXXX-debug.log
-@azure/keyvault-secrets ( ? seconds)
+@azure/eslint-plugin-azure-sdk (? seconds)
+>>> @azure/eslint-plugin-azure-sdk
+tsc -p tsconfig.build.json && prettier --write dist/**/*.{js,json,md}
+../../temp/node_modules/.pnpm/@typescript-eslint/experimental-utils@X.X.X_eslint@X.X.X+typescript@X.X.X/node_modules/@typescript-eslint/experimental-utils/dist/ts-eslint/Linter.d.ts:6:11 - error TS2300: Duplicate identifier 'Linter'.
```

Comment:
Dunno. Don't think the build has changed much recently... this does look like a normal declaration file error, rather than some kind of build conflict (although, yes, installed at a strange looking path thanks to pnpm)